tasks/section3.yml

```diff
@@ -351,15 +422,19 @@
       - rule_3.4.5
 
 - name: "SCORED | 3.4.5 | PATCH | Ensure permissions on /etc/hosts.deny are 644"
-  command: /bin/true
+  file:
+      dest=/etc/hosts.deny
+      owner=root
+      group=root
+      mode=0644
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md